### DISCUSSION TITLE
VoCo intergration?

### DISCUSSION THREAD

**Discussion by Ratteler** [tenacityteam/tenacity-legacy#535] [Ideas]
Several years ago Adobe teased VoCo with AI features that would let users sample voices and then generate performances in that voice. It's clear that Audacity is not going to implement such a feature since their new owners are two busy trying to collect data to sell.

Maybe Tenacity could work on a much needed FOSS version, and make it a distinguishing feature from Audacity.
There is a shit web video editing group call Descript that is offering that for $30.
I would MUCH rather Patreon a FOSS developer that amount.
Since Adobe has their thumb up their ass, they haven't produced VoCo in 4 years, which means you could even beat them to "market".

**Comment by poyribn** [comment]
This sounds good.
If FOSS editing software adds professional features like this, that have never been implemented in "actual" "professional" software,
it'll make FOSS software look much more legitemate to professionals
instead of just remaining "the no-budget alternative for non-professionals".

I don't share this view: To me, FOSS editing software is almost always superior to their "sold for professionals" counterparts (at least if the "professional" software comes from Adobe)
because it's made and continually improved by its actual users
instead of arbitrarily changed (often for the worse) by a clueless team of business boomers.
But I know that FOSS outsiders don't share this view (hopefully only yet).

**Comment by poyribn** [comment]
I just realised that,
because of the very high complexity, and the very advanced infrastructure needed, this would not only have to be financed by a huge sum of money:
It would also have to be its very own application & project, entirely separate from
Tenacity Audio Editor.
